architecture.md

```diff
@@ -0,0 +1,75 @@
+## Overview
```

Comment:
Auf main ist die Datei mittlerweile in ./docs/ARCHITECTURE.md

Bitte rebase auf mindestens https://github.com/internet4refugees/beherbergung/commit/6c08b14a279dd16862be2e92c3c00f42b40395c2

Wenn du dabei Hilfe brauchst, sag gerne bescheid.